
current GPU: NVIDIA A40
0it [00:01, ?it/s]
Traceback (most recent call last):
  File "training.py", line 189, in <module>
    main()
  File "training.py", line 184, in main
    train(model, train_loader, val_loader, device, config)
  File "training.py", line 43, in train
    prediction = model(data)
  File "/u/home/zhousiy/.conda/envs/BodyMeshGNN/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/data2/practical-wise2223/MeshBody/BodyMeshGNN/model.py", line 320, in forward
    x = self.gnn(x, edge_index)
  File "/u/home/zhousiy/.conda/envs/BodyMeshGNN/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/data2/practical-wise2223/MeshBody/BodyMeshGNN/model.py", line 278, in forward
    x = batch_layer(x)
  File "/u/home/zhousiy/.conda/envs/BodyMeshGNN/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/u/home/zhousiy/.conda/envs/BodyMeshGNN/lib/python3.8/site-packages/torch/nn/modules/batchnorm.py", line 171, in forward
    return F.batch_norm(
  File "/u/home/zhousiy/.conda/envs/BodyMeshGNN/lib/python3.8/site-packages/torch/nn/functional.py", line 2450, in batch_norm
    return torch.batch_norm(
RuntimeError: running_mean should contain 128 elements not 32